.flake8

```diff
@@ -0,0 +1,5 @@
+[flake8]
+ignore = E203, E266, E501, W503, F403, F401
+max-line-length = 79
```

Comment:
Why E203 is ignored?
Also, you define line length to be 79 then E501 should not be ignored and this line can be removed?
I recomment not to ignore F403 too and remove all wild card imports.

You should add comments regarding each ignore rule about what they do

Reply:
If line length = 88 is fine then I can just remove these, as black check for line length = 88 by default.